packages/babel-preset-default/index.js

```diff
@@ -1,22 +1,26 @@
-const env = process.env.BABEL_ENV || process.env.NODE_ENV;
-const isTestEnv = env === 'test';
+module.exports = function( api ) {
+	const isTestEnv = api.env() === 'test';
 
-module.exports = {
-	presets: [
-		! isTestEnv && [ require( 'babel-preset-env' ), {
-			modules: false,
-			targets: {
-				browsers: [ 'extends @wordpress/browserslist-config' ],
-			},
-		} ],
-		isTestEnv && [ require( 'babel-preset-env' ) ],
-	].filter( Boolean ),
-	plugins: [
-		require( 'babel-plugin-transform-object-rest-spread' ),
-		[ require( 'babel-plugin-transform-react-jsx' ), {
-			pragma: 'createElement',
-		} ],
-		require( 'babel-plugin-transform-async-generator-functions' ),
-		! isTestEnv && require( 'babel-plugin-transform-runtime' ),
-	].filter( Boolean ),
+	return {
+		presets: [
+			! isTestEnv && [ '@babel/preset-env', {
+				modules: false,
+				targets: {
+					browsers: [ 'extends @wordpress/browserslist-config' ],
+				},
+				useBuiltIns: 'usage',
+			} ],
+			isTestEnv && [ '@babel/preset-env', {
+				useBuiltIns: 'usage',
+			} ],
+		].filter( Boolean ),
+		plugins: [
+			'@babel/plugin-proposal-object-rest-spread',
```

Comment:
I _think_ these can be shortened to e.g. `@babel/proposal-object-rest-spread`. Maybe less explicit. Haven't confirmed.

Reply:
Yes, I think they now discourage it though 🤷‍♂️